docs/_getting-started/index.md

```diff
@@ -4,197 +4,311 @@ permalink: docs/getting-started
+- Snap (Default): Same functionality as fast, but with a faster algorithm.
+- Light: Downloads all block headers, block data, and verifies some randomly.
 
-Create two accounts with the `clef newaccount` command,
-set a password for each of them, and note the public address for each.
+For this tutorial, you will use a `light` sync:
 
-![Create new account command](../../static/images/clef-account-new.gif)
+### Prerequisites:
 
-Clef outputs a debug message about `Failed to reload keystore contents`, we fix that in a later step.
+- Curl experience 
+- Command line
+- Basic Blockchain Knowledge
 
-## Start Geth
+## Step 1: Open Terminal
+You will need your system terminal to run the commands for this tutorial.
 
-### Networks
+![Create new account command](../../static/images/open_terminal.png){:width="70%"}
 
-You can connect a Geth node to several different networks using the network name as an argument.
-These include the main Ethereum network, [a private network](getting-started/private-net) you create,
-and three test networks that use different consensus algorithms:
+### Steps to open your system terminal:
+**Mac book:**
+Press the Command + Space button on your keyboard to open spotlight search, type terminal, and hit return.
 
--   **Ropsten**: Proof-of-work test network
--   **Rinkeby**: Proof-of-authority test network
--   **Görli**: Proof-of-authority test network
+**Windows:**
+Type cmd in the search box, press Enter to open the Command Prompt shortcut highlighted, and then hit the enter button. 
 
-For this guide, we use the Görli network. The default port is 30303, so you need to enable at least
-outgoing access from your node to that port.
+**Linux:**
+To quickly open a Terminal window at any time, press Ctrl+Alt+T.
 
-### Sync modes
+## Step 2: Create accounts
 
-You can start Geth in one of three different sync modes using the `--syncmode "<mode>"`
-argument that determines what sort of node it is in the network.
+use the command below to create an account 
+> **Note:** you will need to create two accounts for this guide
 
-These are:
+```shell
+clef newaccount --keystore geth-tutorial/keystore
+```
+
+it will give you the result below:
 
-- **Full**: Downloads all blocks (including headers, transactions, and receipts) and
-generates the state of the blockchain incrementally by executing every block.
-- **Snap** (Default): Same functionality as fast, but with a faster algorithm.   
-- **Light**: Downloads all block headers, block data, and verifies some randomly.
+![Create new account command](../../static/images/create_account.png){:width="70%"}
 
-For this tutorial, we use a `light` sync:
+Enter “ok” and hit the enter key. Next, the system will request the below action.
+Please enter a password for the new account to be created (attempt 0 of 3)
+Enter your desired password and hit the enter key to get the result below:
 
+![Create new account command](../../static/images/createaccount2.png){:width="70%"}
```

Comment:
Note: the attribute syntax `{:width=70%}` does not work in GitHub Markdown. You can just remove it.

Reply:
It does work in Jekyll, but I think it's not needed.

Reply:
It works well in Jekyll because without it the images are big. I will remove it tomorrow